spec/views/teams/show.html.erb_spec.rb

```diff
@@ -11,4 +11,34 @@
     expect(rendered).to have_content(@team.kind_of_sport, count: 1)
     expect(rendered).to have_content(@team.description, count: 1)
   end
+
+  it " shows assign ownership button for mere members" do
+    user = FactoryBot.create :user    
+    @team.members << user
+    render
+    rendered.should have_content('Als Captain hinzufügen')
+  end
+
+  it "does not show delete ownership button for mere members" do
+    user = FactoryBot.create :user    
+    @team.owners = []
+    render
+    rendered.should_not have_content('Als Captain löschen')
+  end
+
+  it "shows delete ownership button for owners" do
+    user = FactoryBot.create :user    
+    @team.members << user
+    @team.owners << user
+    render
+    rendered.should have_content('Als Captain löschen')
+  end
+
+  it "does not show assign ownership button for owners" do
+    user = FactoryBot.create :user    
+    @team.members << user
+    @team.owners << user
```

Comment:
If the factory for the `team` model would change and create a team with some members that are no team owners, this test would fail. Thus, I would replace this line by 

```ruby
@team.owners = @team.members
```

This would make sure that every member of the team is also an owner. By this means, there would definitely be no assign_ownership button.